types/angular-animate/index.d.ts

```diff
@@ -2,7 +2,7 @@
 // Project: http://angularjs.org
 // Definitions by: Michel Salib <https://github.com/michelsalib>, Adi Dahiya <https://github.com/adidahiya>, Raphael Schweizer <https://github.com/rasch>, Cody Schaaf <https://github.com/codyschaaf>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.3
```

Comment:
I've bumped `@types/angular` to TypeScript 2.3 before and people complained. I don't consider TypeScript 2.3 to be a "safe" requirement. It doesn't seem like there's enough benefit in this case to justify raising the requirement.